rpxy-bin/src/config/toml.rs

```diff
@@ -13,6 +13,7 @@ pub struct ConfigToml {
   pub listen_port: Option<u16>,
   pub listen_port_tls: Option<u16>,
   pub listen_ipv6: Option<bool>,
+  pub tls_redirection_port: Option<u16>,
```

Comment:
I think if `listen_port_tls` is `None`, `tls_redurection_port` should not be `Some`. So we need some assertion.

Reply:
Sounds like a super edge case, but maybe somebody uses rpxy just to redirect but not actually proxy. But maybe better to disallow, don't know